src/file-1.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonarcloud-github-static-resources/v1/common/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:46079/organizations/org5/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:46079/project/issues?id=sample&pullRequest=6992&issues=AWleNiA7Eo9FcZJAz4IY&open=AWleNiA7Eo9FcZJAz4IY)

[ISSUE_KEY]: AWleNiA7Eo9FcZJAz4IY